partial/pandas/_typing.pyi

```diff
@@ -104,7 +104,7 @@ np_ndarray_int64 = npt.NDArray[np.int64]
 np_ndarray_bool = npt.NDArray[np.bool_]
 np_ndarray_str = npt.NDArray[np.str_]
 IndexType = Union[slice, np_ndarray_int64, Index, List[int], Series[int]]
-MaskType = Union[Series[bool], np_ndarray_bool, Sequence[bool]]
```

Comment:
This change doesn't make sense to me. A List is a Sequence, and Sequence is preferred when typing args. Where you having issues with the way it was?

Reply:
`Sequence` includes `Tuple`, and we have to differentiate those cases in pandas.

If you do `df.iloc[3,4]`, the argument is a tuple, and returns a scalar.  If you do `df.iloc[[3,4]]`, the argument is a list, and returns a `DataFrame`.  `Sequence` matches both cases.  Also, `Sequence` matches a `str`, which is painful in other contexts.

Reply:
Got it, thanks for clarifying. I think when I created these a lot of the time the docs said things like "list-like" and those I went and made Sequence.